Parse/PFSubclassing.h

```diff
@@ -36,13 +36,13 @@ NS_ASSUME_NONNULL_BEGIN
  Constructs an object of the most specific class known to implement `+parseClassName`.
 
  This method takes care to help `PFObject` subclasses be subclassed themselves.
- For example, `PFUser.+object` returns a `PFUser` by default but will return an
+ For example, `PFUser.+instantiateObject` returns a `PFUser` by default but will return an
  object of a registered subclass instead if one is known.
  A default implementation is provided by `PFObject` which should always be sufficient.
 
  @return Returns the object that is instantiated.
  */
-+ (instancetype)object;
++ (instancetype)instantiateObject;
```

Comment:
I tried removing it but it's actually used by internal PFInstallation controller somewhere so it will need to be moved to either a private header or a private interface of PFInstallation at least.

I do not use it myself and don't have enough experience to assess whether it's safe to remove it from the interface, but from what I could gather studying the docs the point was to register custom subclasses of PFInstallation and PFUser before initializing Parse API and getting the custom subclasses returned from the SDK, although I found it only being used for PFInstallation in fact...